app/src/organisms/Devices/ProtocolRun/utils/getLabwareDefinitionUri.ts

```diff
@@ -7,13 +7,14 @@ export function getLabwareDefinitionUri(
   labware: ProtocolFile<{}>['labware'],
   labwareDefinitions: ProtocolFile<{}>['labwareDefinitions']
 ): string {
-  const labwareDefinitionId = labware[labwareId].definitionId
-  if (labwareDefinitionId == null) {
+  //  @ts-expect-error
+  const labwareDefinitionUri = labware[labwareId].definitionUri
```

Comment:
This is causing me a bunch of problems. I think it should be  this instead:

```suggestion
  const labwareDefinitionUri = `${labware[labwareId].definitionUri}_id`
```

But if I make the change, a few tests start failing because none of the fixtures have `_id` attached to the end of the the `labwareId`(example test that fails with this change is `createSnippet.test`